site/tutorials/snippets/17/install/istio.md

```diff
@@ -0,0 +1,29 @@
+
+## Install Istio
+Duration: 10:00
+
+Download the Istio command line tool by [following the official instructions](https://istio.io/latest/docs/setup/install/) or by executing the following steps.
+
+<!-- command -->
+```
+curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.12.1 sh -
```

Comment:
Not sure if we also want to update the Istio version since 1.12 is EOL (https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases)

Reply:
I changed this to the newest stable version, `1.14.2`.